### DISCUSSION TITLE
How to find the list of repos I am Watching?

### DISCUSSION THREAD

**Discussion by subidit** [community/community#81652] [Other Feature Feedback, Questions, & Ideas] [answered]
I had clicked on the watch button on some repos that I didn't star or fork. Is there any link where I can find such repos. I remember selecting the watch button for getting notified for any update, but I never got any such notification.

**Comment by ananuness** [comment, marked as answer]
1. You can go to notifications
![image](https://github.com/community/community/assets/51466624/90fb5f26-1db8-40ea-aef1-577592d25f09)

2. Go to the left bottom and click on "Manage notifications" and click Watched repositories.
![image](https://github.com/community/community/assets/51466624/073a26b1-f48b-4a84-97b4-b6257cd15596)

Hope it helps!

**Reply by subidit** [reply]
Got it. Thanks.

**Reply by druid49** [reply]
Why in the heck wouldnt this be right on your dashboard?  theres a reason i flagged it.... so i wouldnt have to click around to find the frickin thing again, cute lil app though

**Comment by anandsuraj** [comment]
Just click on this: https://github.com/watching
